service/rtc/logger.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2022-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package rtc
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/v6/shared/mlog"
+	"github.com/pion/logging"
+)
+
+type pionLogger struct {
+	log mlog.LoggerIFace
+}
+
+func newPionLeveledLogger(log mlog.LoggerIFace) logging.LeveledLogger {
+	return &pionLogger{
+		log: log,
+	}
+}
+
+func (log *pionLogger) Trace(msg string) {
+	log.log.Trace(msg)
+}
+
+func (log *pionLogger) Tracef(format string, args ...interface{}) {
+	log.log.Trace(fmt.Sprintf(format, args...))
+}
+
+func (log *pionLogger) Debug(msg string) {
+	log.log.Trace(msg)
+}
+
+func (log *pionLogger) Debugf(format string, args ...interface{}) {
+	log.log.Trace(fmt.Sprintf(format, args...))
+}
```

Comment:
Purposely downgrading these to trace since they are pretty spammy and we tend to default to DEBUG level for most operations.

Reply:
This is instead of changing the debugs -> trace for those that we actually just want traces of? (rather than removing the debug level)

Reply:
If someone needs trace level logs they can still get them by changing the level in config. Also they can always run either Mattermost (if running with plugin) or `rtcd` with `PION_LOG_TRACE=ice` to get them as well.